WordPress/Classes/ViewRelated/Post/WPPostViewController.m

```diff
@@ -1155,10 +1151,13 @@ - (UIBarButtonItem*)cancelChevronButton
         return _cancelChevronButton;
     }
     
-    WPButtonForNavigationBar* cancelButton = [self buttonForBarWithImageNamed:@"icon-posts-editor-chevron"
-                                                                        frame:NavigationBarButtonRect
-                                                                       target:self
-                                                                     selector:@selector(cancelEditing)];
+    UIImage *image = [UIImage imageNamed:@"icon-posts-editor-chevron"];
```

Comment:
Hmm, maybe we should use the original asset in this case, instead of the gridicon.